tests/hello/hello_update_test.py

```diff
@@ -0,0 +1,23 @@
+import uuid
+
+from temporalio.client import Client, WorkflowExecutionStatus
+from temporalio.worker import Worker
+
+from hello.hello_update import GreetingWorkflow
+
+
+async def test_update_workflow(client: Client):
```

Comment:
Sorry it took so long to get back to this. There is an error in our time-skipping test server wrt updates. Can you skip it?

```suggestion
async def test_update_workflow(client: Client(client: Client, env: WorkflowEnvironment):
    if env.supports_time_skipping:
        pytest.skip("Time-skipping test server currently has issue with update: https://github.com/temporalio/sdk-java/issues/1903")
```

Reply:
Done